src/components/AssetInput/AssetInput.stories.tsx

```diff
@@ -117,40 +68,169 @@ PlaceholderLibraryOnly.argTypes = {
 export const Image = Template.bind({});
 
 Image.args = {
-    asset: {
-        source: "upload",
-        name: "foo",
-        size: 2000,
-        type: "image",
-        extension: "JPG",
-        src: "https://picsum.photos/200/300",
-    },
+    assets: [
+        {
+            source: "upload",
+            name: "foo",
+            size: 2000,
+            type: "image",
+            extension: "JPG",
+            src: "https://picsum.photos/200/300",
+        },
+    ],
     actions,
 };
 
 export const Audio = Template.bind({});
 
 Audio.args = {
-    asset: {
-        source: "library",
-        sourceName: "Foobar",
-        name: "foo",
-        extension: "MP3",
-        size: 2000,
-        type: "audio",
-    },
+    assets: [
+        {
+            source: "library",
+            sourceName: "Foobar",
+            name: "foo",
+            extension: "MP3",
+            size: 2000,
+            type: "audio",
+        },
+    ],
     actions,
 };
 
 export const Icon = Template.bind({});
 
 Icon.args = {
-    asset: {
-        source: "library",
-        sourceName: "Foobar",
-        name: "foo",
-        type: "icon",
-        icon: <IconIcons />,
-    },
+    assets: [
+        {
+            source: "library",
+            sourceName: "Foobar",
+            name: "foo",
+            type: "icon",
+            icon: <IconIcons />,
+        },
+    ],
     actions,
 };
+
+const TemplateMultiInput: Story<AssetProps> = (args) => {
+    return <AssetInput {...args} />;
+};
+
+export const TwoAssets = TemplateMultiInput.bind({});
+
+TwoAssets.args = {
+    assets: [
+        {
+            name: "foo1",
+            size: 2000,
+            type: "image",
+            extension: "JPG",
+            src: "https://picsum.photos/100/150",
+            source: "library",
+            sourceName: "",
+        },
+        {
+            source: "library",
+            sourceName: "Foobar",
+            name: "foo",
+            type: "icon",
+            icon: <IconIcons />,
+        },
+    ],
+    numberOfLocations: 1,
+    actions,
+};
+
+export const OnlyImageAssets = TemplateMultiInput.bind({});
+
+OnlyImageAssets.args = {
+    assets: [
```

Comment:
Maybe you could move and export the EXAMPLE_IMAGES from the test file into a separate file and reuse them for the stories

Reply:
I once tried – didn't work. Now tried again – works smoothly. Don't know what I tried back than 😂